ledger/src/shredder.rs

```diff
@@ -260,6 +260,7 @@ impl Shredder {
                 .into_iter()
                 .zip(next_code_index)
                 .flat_map(|(shreds, next_code_index)| {
+                    #[allow(deprecated)]
```

Comment:
why do we continue using it if it is deprecated?

Reply:
This function will get removed soon. Most functions in that file need to go.

Reply:
Why do you want to deprecate it now in this case?

Reply:
So people can not accidentally use it via public API.